src/OSPSuite.Core/Domain/UnitSystem/Unit.cs

```diff
@@ -21,14 +26,43 @@ public class Unit
       /// <summary>
       ///    Name of unit. Should be unique in the dimension containing this unit
       /// </summary>
-      public virtual string Name { get; }
+      public virtual string Name
+      {
+         get => _name;
+         internal set
+         {
+            _name = value;
+            addDefaultUnitSynonyms();
+         }
+      }
+
+      private void addDefaultUnitSynonyms()
+      {
+         if(!_name.Contains("�"))
```

Comment:
@Yuri05 I think this is better like this for our µ issue. 
We still support this for other use cases (e..g loading from XML) but i don't think we should have all those options directly in the XML. It's just too redundant

Reply:
yep, good idea